umh.docs.umh.app/content/en/docs/features/datainfrastructure/network-view.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Network Monitoring
+menuTitle: Network Monitoring
+description: |
+  Monitor all instances within the Unified Namespace at a glance.
+weight: 2000
+draft: true
+---
+
+The Network Monitoring feature of the United Manufacturing Hub visualizes the instance's
+network within the Unified Namespace.
+
+## When should I use it?
+
+Your infrastructure might consist of multiple UMH instances. This feature offers
+an intuitive view for monitoring the entire network of all instances and their
+network devices, allowing you to detect any issues within the infrastructure.
+
+## What can I do with it?
+
+The network view offers a graph where instances are usally grouped by the ISA-95 schema,
+these instances form a tree-like structure, which there can be multiple of, making it
+easy to visualize the whole network and the relationships between them. There can be
+instances that don't fit into the ISA-95 schema, these appear at the root of the graph,
+without any parent nodes.
+
+![Network view](/images/features/network-view/networkViewDemo.png?width=80%)
+
+It might seem like there's a lot going on in the image above, but we can break it down into 3
+different types of nodes:
+
+1. **Location**: These are the nodes to visualize the ISA-95 schema and are colored with a
+   black background with a slight gray border. In the image above, one can be seen with the
+   label `demo-microchip-enterprise`, representing the enterprise level.
+2. **UMH Instance**: These are the nodes that represent a UMH instance and are colored with
+   a light gray background. In the image above, several instances can be seen, such as `Cologne`,
+   living at the **site** level, as well as non ISA-95 instances like `Instance1`, which doesn't
+   fit into any location level.
+3. **Network Device**: These are the nodes that represent a network device and are colored with
+   a slight darker gray background. Network devices can only be descendants of UMH instances, and
```

Comment:
children instead of descendants?

Reply:
I mean it's whatever, both terms are probably fine. If it's preferred, then i'll just change it.